tests/draft4/format.json

```diff
@@ -1,216 +1,75 @@
 [
     {
-        "description": "validation of e-mail addresses",
-        "schema": {"format": "email"},
+        "description": "validation of string formats",
+        "schema": {
+            "allOf": [
```

Comment:
I think I like the previous way more personally if I'm honest.

Or, I don't really have a strong opinion either way on putting these in one place or in each format, but I slightly prefer we don't complicate the schema -- i.e. use `allOf`.

(I think basically the simpler each schema can be in the test suite, the better -- we don't mean to test interaction between allOf and format in these tests, so it's better if we don't use it, essentially)

Also, previously someone would get one test per format if you map each test case to a test case in your language's testing framework. But now it's one test for all formats, which likely will fail in a more obscure way if one format is broken.

Reply:
Consistency is good though -- e.g. it'd be nice certainly to have all of these kinds of tests looking the same way (using the same description, etc.).